packages/react-components/react-popover/src/components/Popover/usePopover.ts

```diff
@@ -75,12 +75,15 @@ export const usePopover_unstable = (props: PopoverProps): PopoverState => {
     refs: [popperRefs.triggerRef, popperRefs.contentRef],
     disabled: !open,
   });
+
+  // only close on scroll for context, or when closeOnScroll is specified
+  const closeOnScroll = open && (initialState.openOnContext || initialState.closeOnScroll);
```

Comment:
```suggestion
  const closeOnScroll = initialState.openOnContext || initialState.closeOnScroll;
```

I would remove `open` from the expression since it does not actually refer to closing on scroll

Reply:
I think the `open` was here to prevent adding the wheel/touchmove event listeners on document when the popup is closed

Reply:
Oh I just saw your other comment on line 86 :) Got it, let me make the change